packages/account-v2/src/components/base/WalletDatePicker/WalletDatePicker.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useEffect, useRef, useState } from 'react';
```

Comment:
This is a reusable component use in different screens should be inside of `deriv-com/ui`